edge-util/src/Microsoft.Azure.Devices.Edge.Util/CertificateHelper.cs

```diff
@@ -208,6 +208,33 @@ public static async Task<(X509Certificate2 ServerCertificate, IEnumerable<X509Ce
             return ParseCertificateResponse(response);
         }
 
+        public static (X509Certificate2 ServerCertificate, IEnumerable<X509Certificate2> CertificateChain) GetServerCertificateAndChainFromFile(string serverWithChainFilePath, string serverPrivateKeyFilePath)
+        {
```

Comment:
Can you add tests for GetServerCertificateAndChainFromFile

Reply:
Added tests